### PR TITLE
Render client version if build json has information about it

### PR DESCRIPTION
**Pull request by vrutkovs**
See https://github.com/projectatomic/osbs-client/issues/477

### CODE REVIEW

File: atomic_reactor/inner.py [hunk 1]
```diff
@@ -297,6 +297,9 @@ def __init__(self, source, image, prebuild_plugins=None, prepublish_plugins=None
 
         self.openshift_build_selflink = openshift_build_selflink
 
+        if client_version:
+            logger.debug("build json was build by osbs-client %s", client_version)
```
Comment (twaugh) [inline atomic_reactor/inner.py]: Actually this should probably say 'built by'

Comment (vrutkovs) [inline atomic_reactor/inner.py]: Oh, right. Fixed

File: tests/test_inner.py [hunk 2]
```diff
@@ -940,3 +941,40 @@ def restore_logger():
...
+        'exit_plugins': []
+    }
+    if has_version:
+        params['client_version'] = VERSION
+
+    workflow = DockerBuildWorkflow(MOCK_SOURCE, 'test-image', **params)
+    workflow.build_docker_image()
+
+    if has_version:
+        assert ("build json was built by osbs-client %s", VERSION) in fake_logger.debugs
```
Comment (lcarva) [inline tests/test_inner.py]: Maybe to catch both cases:

```python
assert (("build json was built by osbs-client %s", VERSION) in fake_logger.debugs) == has_version
```

Comment (vrutkovs) [inline tests/test_inner.py]: Ah, that's a good idea. Fixed

**Pull request merged**